### DISCUSSION TITLE
[slider] how change the label property when the value is selected

### DISCUSSION THREAD

**Discussion by yyaanniicckk** [fomantic/Fomantic-UI#2392] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello,
I would like to know if it is possible to know which value is selected in order to change its CSS parameters (as on the example).
Thanks in advance.

![image](https://user-images.githubusercontent.com/108870769/177752687-2f2f8758-90ea-45e4-b939-b0735bc8d4e7.png)

**Comment by jamessampford** [comment, marked as answer]
Use the `onChange` callback to get the current value then use some JavaScript magic to apply/remove CSS as need be

https://fomantic-ui.com/modules/slider.html#/settings

**Comment by yyaanniicckk** [comment]
Yeah, that's what I did. It wasn't easy but I did it.
